taxonomies/ingredients.txt

```diff
@@ -6609,6 +6609,7 @@ es:aceite de girasol prensado en frio
 fr:huile de tournesol pressée à froid
 nl:koudgeperste zonnebloemolie
 pt:óleo de girassol prensado a frio
+carbon_footprint_fr_foodges_ingredient:fr:Huile Tournesol
```

Comment:
Is there a more generic one ? fr:huile de tournesol 
The carbon footprint might change due to manufacturing